### ISSUE TITLE
Remove the Remember Me checkbox, and have the login remembered by default

### ISSUE THREAD

**Issue by stuzart** [seek4science/seek#1095]
Currently the checkbox is unticked, leading to the likelihood of forgetting to check it when logging in, meaning you will be logged out again after a short while.

It is becoming increasing common for sites to no longer provide this option, and also automatically remember you. Newer users may not even be familiar with what the option means.

**Comment by OlgaKrebs** [comment]
I would leave checkbox, but activated by default. For users using shared computers, the option to activate time out is important

**Comment by stuzart** [comment]
> I would leave checkbox, but activated by default. For users using shared computers, the option to activate time out is important

I tend to agree with @OlgaKrebs - this is the one reason it could be best to leave it. Although if using a shared computer you should still be logging out, but this at least does a give a bit of extra safety
